tests/test_settings.py

```diff
@@ -72,12 +72,61 @@ def test_cycle_value_loop(hlwm):
 
```

Comment:
Can you add a test for an invalid layout algorithm string? E.g. 
```set default_frame_layout foobar```

Reply:
Ah there already is one with ```test_default_frame_layout_value_negative```.
Maybe we can rename the test? It doesn't fit that nicely anymore IMHO?

Reply:
I've renamed it. (And I'm testing for `-1` and `foobar`.

Reply:
Thanks!